Documentation/Publishing.md

```diff
@@ -25,4 +25,15 @@ To sign and publish the artifacts run the following Gradle command:
 We exclude the client-rest archives as these are not a Java package.
 
 ## 4. Release
-You can then release the deployment to the Central Repository following the steps on the Sonatype website  [here](https://central.sonatype.org/publish/release/). We are on the legacy host so make sure to use https://oss.sonatype.org/. Only attempt to Close the staging repository once the upload has finished.
\ No newline at end of file
+You can then release the deployment to the Central Repository following the steps on the Sonatype website  [here](https://central.sonatype.org/publish/release/). We are on the legacy host so make sure to use https://oss.sonatype.org/. Only attempt to Close the staging repository once the upload has finished.
+ 
+## 5. Create github release and tag
+
+1. Go to [/releases](https://github.com/google/data-transfer-project/releases);
+2. Click 'Draft a new release';
+3. fill in the title and create a tag (the tag must have a `v` prefix, the title mustn't);
```

Comment:
I'm imagining we'd prefer semver releases - should we just say that explicitly?

Reply:
Or maybe call out that things are still in flux[^1] and say something _vaguely_ about semver; eg: _"step 0: start a discussion in maintainers chat about setting a new release (until the process is better documented); step 3: pick a new release version name by referring to our previous release, according to semver.org values (major, minor, patch)"_

[^1]: I think you said you were going to reachout in maintainers room with a doc or something? SGTM!

Reply:
Added notes on versioning as a separate section. What do you think about the updated version?

Reply:
Discussed in the call on Monday